portfolio/src/main/webapp/assets/script.js

```diff
@@ -171,11 +171,18 @@ function createListElement(text) {
 }
 
 /**
- * Fetches comments from server and adds to forum page.
+ * Fetches comments from server and adds to forum page with querySize that is specified
+ * by button. Default is 5 comments showing.
  */
 
 function getComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  var querySize = document.getElementById("comments-num").value;
+  if (querySize == undefined) {
+    querySize = 5;
```

Comment:
Can you define this as a constant above:

const defaultNumComments = 5;

and then reference the const. This is just more readable.